plugins/modules/dcnm_interface.py

```diff
@@ -4585,6 +4605,13 @@ def dcnm_intf_send_message_to_dcnm(self):
         path = self.paths["GLOBAL_IF"]
         for payload in self.diff_create:
 
+            # The 'int_fabric_loopback_11_1' policy is a special policy for
+            # managing fabric loopback interfaces.  These interfaces are created
+            # at fabric creation time and can only be manged by appening
+            # '/pti?isMultiEdit=false' to the path.
+            if payload.get('policy') == 'int_fabric_loopback_11_1':
+                path = path + '/pti?isMultiEdit=false'
+
```

Comment:
I remember we discussed regarding POST and PUT for these fabric interfaces. You mentioned that we should always use POST for these interfaces even if this interface already exist. I don't see the code for forcing POST even if a fabric interface already exist.

Reply:
It turns out that `PUT` does work with `/pti?isMultiEdit=false`.  I had not tested it with this setting so looks like we are ok.

Reply:
**Update:** After more testing I discovered that if the module uses the correct template for the loopback interfaces I don't need to add /pti?isMultiEdit=false to the path. It's sufficient to just use PUT